internal/app/machined/internal/server/v1alpha1/v1alpha1_server.go

```diff
@@ -2044,6 +2048,97 @@ func (s *Server) GenerateClientConfiguration(ctx context.Context, in *machine.Ge
 	return reply, nil
 }
 
+// PacketCapture performs packet capture and streams the pcap file.
+//
+//nolint:gocyclo
+func (s *Server) PacketCapture(in *machine.PacketCaptureRequest, srv machine.MachineService_PacketCaptureServer) error {
+
+	log.Printf("in PacketCapture")
```

Comment:
debug

Reply:
yeah, lots of debug still :) it's heavy WIP